rllib/callbacks/utils.py

```diff
@@ -60,6 +60,9 @@ def on_workers_recreated(self, **kwargs) -> None:
         def on_env_runners_recreated(self, **kwargs) -> None:
             pass
 
+        def on_offline_eval_runners_recreated(self, **kwargs) -> None:
```

Comment:
remove these: Old API stack only!

The new API stack allows specifying a simple list of callbacks classes in the config w/o having to go through this helper class.

Reply:
I added this later in response to some callback tests failing and specifically telling that these callbacks were missing. I think it was `test_callback_old_api_stack.py`